crates/autopilot/src/run_loop.rs

```diff
@@ -289,6 +293,8 @@ impl RunLoop {
                 competition_table,
             };
 
+            Metrics::post_processed(start.elapsed());
```

Comment:
Afaict, there is no async work here so I'd be extremely surprised if this took more than a few ms. Probably not worth measuring (let's combine it with the storing)